usaspending_api/awards/management/commands/loadfinancialassistance.py

```diff
@@ -0,0 +1,189 @@
+        # none of those really look like enough to identify an agency
+        # how to identify the funding agency?
+
+        value_map = {
+            "data_source": "USA",
+            "submitted_type": "C",  # ?? For CSV?
+            "recipient": self.get_or_create_recipient,
+            "award": self.get_or_create_award,
+            "place_of_performance":
+            lambda row: h.get_or_create_location(row, location_mapper_fin_assistance_principal_place),
+            "awarding_agency": self.get_awarding_agency,
+            "assistance_type": lambda row: h.up2colon(row['assistance_type']),
+            "record_type": lambda row: int(h.up2colon(row['record_type'])),
+            "action_type": lambda row: h.up2colon(row['action_type']),
+            "action_date":
+            lambda row: h.convert_date(row['obligation_action_date']),
+            "last_modified_date":
+            lambda row: h.convert_date(row['last_modified_date']),
+            "transaction_number":
+            lambda row: self.parse_first_character(row['transactionnumber']),
+            "solicitation_identifier":
+            lambda row: self.parse_first_character(row['solicitationid']),
+            "submission": subattr
+        }
+
+        loader = ThreadedDataLoader(
+            FinancialAssistanceAward, field_map=field_map, value_map=value_map)
+        loader.load_from_file(options['file'][0])
+
+    def get_or_create_award(self, row):
+        fain = row.get("federal_award_id", None)
+        award = Award.get_or_create_summary_award(fain=fain)
+        return award
+        # TODO: really don't know how to detect parent awards
+
+    def recipient_flags_by_type(self, type_name):
+        """Translates `type_name` to a T in the appropriate flag value.
+
+        Assumes that all other flags are null.
+        """
+
+        flags = {}
+
+        mappings = {
+            'State government': 'us_state_government',
+            'County government': 'county_local_government',
+            'City or township government': 'city_township_government',
+            # or city_local_government, county_local_government, municipality_local_government ?
+            'Special district government': 'special_district_government',
+            'Independent school district': 'school_district_local_government',
+            'State controlled institution of higher education':
+            'educational_institution',
+            'Indian tribe': 'us_tribal_government',
+            # or indian_tribe_federally_recognized ?
+            'Other nonprofit': 'nonprofit_organization',
+            'Private higher education': 'educational_institution',
+            'Individual': 'individual',
+            'Profit organization': 'for_profit_organization',
+            'Small business': 'small_business',
+            # should the more specific contract flags for small businesses also set this to Y?
+            'All other': '',
+        }
+        flag = mappings.get(type_name, "")
```

Comment:
Are these guaranteed to have the same capitalization/wording each time? (I hope so...)